src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -116,15 +142,36 @@ def validate_teamRoles(self, teamRoles) -> list[tuple[Team, str]]:
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Organizations"])
 class OrganizationMemberIndexEndpoint(OrganizationEndpoint):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
-        "POST": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
+        "POST": ApiPublishStatus.PUBLIC,
     }
     permission_classes = (MemberAndStaffPermission,)
     owner = ApiOwner.ENTERPRISE
 
+    @extend_schema(
+        operation_id="List an Organization's Members",
+        parameters=[
+            GlobalParams.ORG_SLUG,
+        ],
+        responses={
+            200: inline_sentry_response_serializer(
+                "ListOrganizationMemberResponse", list[OrganizationMemberResponse]
+            ),
+            401: RESPONSE_UNAUTHORIZED,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=OrganizationMemberExamples.LIST_ORG_MEMBERS,
+    )
     def get(self, request: Request, organization) -> Response:
+        """
+        List all organization members.
+
+        Response includes pending invites as long as they're already approved.
```

Comment:
a bit confused on this wording, is this referring to members who were re-invited?

Reply:
As soon as someone gets invited, even if they didn't accept the invite they become "member". I'll try to explain better.